cloudbackup_mon.sh

```diff
@@ -82,19 +96,21 @@ fi
 
 # Get username and agentid
 username=`cat /etc/driveclient/bootstrap.json | grep Username | awk '{print $3}' | sed -e 's/"//g' -e 's/,//g'`
-agentid=`cat /etc/driveclient/bootstrap.json | grep AgentId | awk '{print $3}' | sed -e 's/,//g'`
+agentid=`cat /etc/driveclient/bootstrap.json | grep AgentId\" | awk '{print $3}' | sed -e 's/,//g'`
 
 # Get token
-token=`curl -s -I -H "X-Auth-Key: $api_key" -H "X-Auth-User: $username" https://auth.api.rackspacecloud.com/v1.0 | grep -vi 'vary:' | grep -i X-Auth-Token |awk {'print $2'}`
+json=`curl --data "{ \"auth\":{ \"RAX-KSKEY:apiKeyCredentials\":{ \"username\":\"${username}\", \"apiKey\":\"${api_key}\" } } }" -H "Content-Type: application/json" -X POST https://auth.api.rackspacecloud.com/v2.0/tokens`
+token=`echo $json | python -m json.tool | grep token -A5 | grep id | cut -d\" -f4`
+url=$(echo $json | python -m json.tool | grep backup | grep $dc | cut -d\" -f4)
```

Comment:
We now get the backup api url from the json output, instead of hardcoding a broken url